tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -197,7 +200,9 @@ trait WebActionsApiBaseTests extends ControllerTestCommon with BeforeAndAfterEac
               annotations ++
                 Parameters("web-export", JsBoolean(true)) ++ {
                 if (requireAuthentication) {
-                  Parameters("require-whisk-auth", JsBoolean(true))
+                  Parameters(
+                    "require-whisk-auth",
```

Comment:
we should probably start pulling all the annotations to an Annotations singleton.